sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/NetAppVault.Serialization.cs

```diff
@@ -69,7 +67,7 @@ internal static NetAppVault DeserializeNetAppVault(JsonElement element)
                     continue;
                 }
             }
-            return new NetAppVault(id, name, type, systemData.Value, vaultName.Value);
+            return new NetAppVault(id, name, type, systemData.Value, name);
```

Comment:
why we change the vaultName to name here? I believe this will break the behaviour on lower api-versions.